packages/client-core/src/common/services/ProjectService.ts

```diff
@@ -99,7 +99,7 @@ export const ProjectService = {
 
   // restricted to admin scope
   uploadProject: async (data: ProjectBuildUpdateItemType) => {
-    const result = await API.instance.client.service(projectPath).update({
+    const result = await API.instance.client.service(projectPath).update('', {
```

Comment:
Shouldnt this be null instead of empty?

Reply:
Id parameter for update method is not nullable, `Argument of type 'null' is not assignable to parameter of type 'Id'`.

`update(id: Id, data: Data, params?: ServiceParams): Promise<Result>;`